types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
@RyanCavanaugh so we change to stripe-v2 and stripe-v3 right? Or just 
stripe/v2
stripe/v3

and they will be published as a @types/stripe/v2 and @types/stripe/v3 right?